kubemarine/kubernetes/__init__.py

```diff
@@ -798,6 +804,33 @@ def get_kubeadm_config(inventory: dict) -> str:
     return f'{kubeadm_kube_proxy}---\n{kubeadm_kubelet}---\n{kubeadm}'
 
 
+def reconfigure_kube_proxy_configmap(control_plane: NodeGroup, mutate_func: Callable[[dict], dict]) -> None:
```

Comment:
This is currently not used, so it is not clear for me why we need to merge it as is. In general I think it is not good to add unused code